README.md

```diff
@@ -120,6 +123,30 @@ need to be provided in order for this action to successfully run using one of th
 | ------------------------- | -------- | ----------- |
 | `imageToDeploy`           | Yes (for this scenario) | The name of the image that has already been pushed to a registry and will be deployed to the Container App by this action. If this image is found in an ACR instance that requires authentication to pull, the `acrName` argument, or the `acrUsername` and `acrPassword` arguments, can be provided to authenticate requests to the ACR instance. |
 
+### Arguments required for using a YAML configuration file
+
+| Argument name             | Required | Description |
+| ------------------------- | -------- | ----------- |
+| `yamlConfigPath`          | Yes (for this scenario) | Full path (on the executing GitHub runner) to the YAML file detailing the configuration of the Container App. |
+
+#### Important note on the YAML configuration file
+
+When providing the `yamlConfigPath`, identity arguments (_e.g._, `containerAppName`, `resourceGroup`) provided

```

Comment:
OK I don't know if it's just me but `identity` argument is a big ambiguous. My first thought was authentication/managed identity stuff. Maybe we could just list the 3-4 arguments that are actually impacted?

Reply:
Yes that's a good point -- let me clear that up a bit for users.